packages/component-library/components/NavigationBar/components/Dropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useLayoutEffect, useRef, useState } from "react"
+
+import classNames from "classnames"
+import { MenuGroup, MenuItemProps } from "../types"
+import MenuItem from "./MenuItem"
+
+const styles = require("./Menu.module.scss")
```

Comment:
Does the `Dropdown` component make exclusive use of the `Menu` styles? 

Wonder if it's worth moving that style code to `Dropdown.module.tsx`. This would prevent downstream changes if things were changed in the `Menu.module.scss` 

Another option might be to compose them from a central spot where `Menu.module.scss` and `Dropdown.module.scss` could `compose` / `@include` them.

Reply:
I was going to separate it out when I do mobile uplift.